creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -8,6 +9,25 @@ const _ = require('lodash')
 // We add a namespace prefix to differentiate internal metrics from those exported by different exporters from the same host
 const NamespacePrefix = 'audius_cn_'
 
+const MetricLabels = Object.freeze({
+  ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM: {
+    // The reason another sync is needed
+    reason_for_additional_sync: [
+      'secondary_progressed_too_slow', // The secondary sync went through, but its clock value didn't increase enough
```

Comment:
agreed, i'm fine with pre-defining, lets pass those const var names around instead of re-defining string? i see this in a few places, ideally would limit it - wdyt? i'm flexible

Reply:
just pushed some changes for this -- how does it look now?

Reply:
looks good! we still re-define the string literal in the job processor but its fine since you have validation

Reply:
oh yea for that one I couldn't think of a way to reference the label name/value via a variable without overcomplicating it but it has the validation like you said

Reply:
yeh its fine